monsteer/qt/steeringWidget/SteeringWidget.h

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2011-2015, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+ *
+ * This file is part of Monsteer <https://github.com/BlueBrain/Monsteer>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#ifndef _SteeringWidget_h_
+#define _SteeringWidget_h_
+
+#include <monsteer/types.h>
+
+#ifndef Q_MOC_RUN
+#  include <monsteer/ui_SteeringWidget.h>
```

Comment:
ouch, this should NOT be included in the header! It renders the pimpl completely useless... Look here for how I solved it in livreGUI: https://github.com/rdumusc/Livre/commit/c5a3d992e0ffe2c389028cd5402e1c5e7d5d2a19#diff-2668ef5a9e57d1e60c23a305c4734c3eR214